cli/bpmetadata/proto/bpmetadata_ui.proto

```diff
@@ -148,11 +148,13 @@ message DisplayVariable {
   }
   repeated AlternateDefault alt_defaults = 21; // @gotags: json:"altDefaults,omitempty" yaml:"altDefaults,omitempty"
 
-  repeated DisplayVariableToggle toggle_using_variables = 22; // @gotags: json:"toggleUsingVariables,omitempty" yaml:"toggleUsingVariables,omitempty"
+  DisplayVariableToggle toggle_using_variable = 22;  // @gotags: json:"toggleUsingVariable,omitempty" yaml:"toggleUsingVariable,omitempty"
+
+  repeated DisplayVariableToggle toggle_using_variables = 23; // @gotags: json:"toggleUsingVariables,omitempty" yaml:"toggleUsingVariables,omitempty"
```

Comment:
@apeabody @njlisi @g-awmalik Changing the field name/number should be fine as this is the exact field names/numbers used within g3.
So syncing this would not make change to g3. PTAL!